truss-chains/truss_chains/code_gen.py

```diff
@@ -326,6 +327,8 @@ def _make_chainlet_dir(
     chainlet_name = chainlet_descriptor.name
     dir_name = f"chainlet_{chainlet_name}"
     chainlet_dir = root / definitions.GENERATED_CODE_DIR / chain_name / dir_name
+    if chainlet_dir.exists():
+        shutil.rmtree(chainlet_dir)
```

Comment:
this will remove the existing generated truss from tmp directory, right?

Reply:
Correct.